### ISSUE TITLE
DataBase issue

### ISSUE THREAD

**Issue by JamesMyer** [WWBN/AVideo#8786]
Please see the attached PIcs , one shows the blank space where the Formate buttons should be , the other the expected buttons , just for example.

Please also see the logs

Have you seen this before ? advise on what to do would be very helpful.

James
![Screenshot 2024-01-10 at 11 13 28](https://github.com/WWBN/AVideo/assets/155841196/4008001d-a066-4a99-8659-459ac1cd6c51)
![Screenshot 2024-01-10 at 11 14 03](https://github.com/WWBN/AVideo/assets/155841196/687ad0b5-9005-48a2-8afe-0e6b7e9bb61d)
![Screenshot 2024-01-10 at 11 17 44](https://github.com/WWBN/AVideo/assets/155841196/75538d49-862b-404c-82e8-8cef525b6c51)

[

**Comment by DanielnetoDotCom** [comment]
your video has only 8 seconds. maybe this video is a mistake.

it will not show buttons if the video has a invalid status

**Comment by JamesMyer** [comment]
Hi, there are other videos that are much longer , same thing

**Comment by DanielnetoDotCom** [comment]
I cannot see your video status, you may need to check what is it in the database

I am not sure what is your issue

**Comment by JamesMyer** [comment]
Hi the status is Active, do you want the logs ?

**Comment by DanielnetoDotCom** [comment]
what exactly is the value for the column videos.status?

**Comment by JamesMyer** [comment]
where would I find that?

**Comment by DanielnetoDotCom** [comment]
directly into the database

**Comment by JamesMyer** [comment]
oh , no idea , would the logs show it?

**Comment by DanielnetoDotCom** [comment]
No, the status, must be checked in the database, directly in the video row. 

but maybe the reason why your status did not update correctly is in the logs

**Comment by JamesMyer** [comment]
are there any services that may need starting?

**Comment by DanielnetoDotCom** [comment]
I do not know what is your issue. I cannot tell you what to do until we findout

**Comment by JamesMyer** [comment]
[avideolog.xlsx](https://github.com/WWBN/AVideo/files/13895515/avideolog.xlsx)

**Comment by JamesMyer** [comment]
I am grateful for any help help you can give.

**Comment by JamesMyer** [comment]
Can anyone help with this please, this is now very urgent

**Comment by DanielnetoDotCom** [comment]
My recommendation for the most effective solution would be to opt for let me make a clean installation. This approach often resolves a variety of underlying issues efficiently.

However, I completely respect your decision if you prefer not to start from scratch. In that case, I'm available to provide a more personalized troubleshooting service. Please note that this service is billed on an hourly basis. While I will make every effort to resolve all issues, I must inform you that I can't guarantee a complete resolution in every scenario, due to the complex nature of technical problems.

If you decide to proceed with hiring my services, you can do so through our marketplace at: [YouPHPTube Marketplace](https://youphp.tube/marketplace/).
